pixi.js.d.ts

```diff
@@ -659,32 +665,32 @@ declare namespace PIXI {
         static TEMP_MATRIX: Matrix;
 
     }
-    export class ObservablePoint {
 
-        constructor(cb: () => any, scope?: any, x?: number, y?: number);
+    class PointLike {
```

Comment:
PointLike isn't an actual object, it's just a shared interface for both Point and ObservablePoint.

In #156 @clark-stevenson thought it would be more clear to use Point|ObservablePoint instead of the PointLike interface.